com.unity.render-pipelines.high-definition/Documentation~/Compositor-User-Guide.md

```diff
@@ -38,18 +45,47 @@ To add a Sub-layer to a Composition Layer, select the Composition Layer and clic
+
+Note that enabling expensive post-processing effects (such as Depth of Field and Bloom) in multiple Sub-layes will negatively impact the run-time performance. Therefore it is always more preferable to run the post-processing in groups of layers when possible, by smartly utilizing the "Clear Alpha" option in the Sub-layer Properties.
+
+### Alpha-based composition and Bloom 
+The general rule is that post-processing will affect only pixels that have an alpha value that is greater than zero. An exception to this rule is the Bloom effect, where the glow that is generated will also spread on pixels with zero alpha. However, only source pixels with an alpha that is greater than zero will contribute to the bloom effect (with a contribution propertional to the alpha value).
+
 ## Camera Stacking
-When you use more than one Sub-layer to specify the content of a Composition Layer, this "stacks" the Sub-layers on top of the same render target. To specify the size and format of the composition, you use the properties of the parent Composition Layer. The Sub-layers inherit the size and format from their parent Composition Layer and you cannot change these properties independently for a particular Sub-layer. This means every stacked Camera/Sub-layer has the same size and format. To change the stacking order, re-arrange the Sub-layers in the [Render Schedule](#render-schedule) section of the Graphics Compositor window.
+When you use more than one Sub-layer to specify the content of a Composition Layer, this "stacks" the Sub-layers on top of the same render target. To specify the size and format of the render target, you use the properties of the parent Composition Layer. The Sub-layers inherit the size and format from their parent Composition Layer and you cannot change these properties independently for a particular Sub-layer. This means every stacked Camera/Sub-layer has the same size and format. 
+
+To change the stacking order, re-arrange the Sub-layers in the [Render Schedule](#render-schedule) section of the Graphics Compositor window. The composition is depth-aware, which means that for layers with opaque surfaces, reordering the Sub-layers will not make any difference in the resulting image, as long as the "Clear Depth" option in the Sub-layer is not selected (therefore depth is preserved between Sub-layers).
+
+Chosing the right order for a Sub-layer is important for elements that don't draw any depth (sunch as UI elements) and generally for getting proper screen space reflections and transparency between layers. As an example, a layer with semi-transparent UI elements that should be rendered on top of the 3D world should appear last in the stacked sub-layers of a Composition Layer.
```

Comment:
Choosing the correct order for a Sub-layer is important for elements that don't draw any depth (such as UI elements) and generally for getting proper screen-space reflections and transparency between layers. For example, a layer with semi-transparent UI elements that should render on top of the 3D world should appear last in the stacked sub-layers of a Composition Layer.